script.js

```diff
@@ -202,8 +202,8 @@ function formValidate(event) {
   } else { error.style.visibility = 'hidden'; }
 }
 form.addEventListener('submit', formValidate);
-const message = document.getElementById('message');
-const userName = document.getElementById('contact-name');
+const message = document.getElementByclass('message');
+const userName = document.getElementByclass('contact-name');
 
 function storeData() {
```

Comment:
- [ ] Well! Your UI looks visually appealing. However, according to the project requirements, when the user changes the content of any input field, the data should be saved to the local storage. Additionally, when the user loads the page, if there is any data stored in the local storage, the input fields should be pre-filled with that data. 🔄

Currently, it seems that this functionality is not working as expected in your project. Please review and update your code to ensure that the data is properly saved to the local storage and that the input fields are correctly pre-filled upon page load. Keep up the good work! 💪🌟

[Portfolio.webm](https://github.com/Nelofarzabi/My_Portfolio/assets/55840999/8f3331f8-3b64-481e-a8ed-5c179fc0ff4d)